librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -300,7 +300,10 @@ commands:
       - name: dcs
         summary: Continue until syscall
         cname: cmd_debug_continue_syscall
-        type: RZ_CMD_DESC_TYPE_OLDINPUT
+        args:
+          - name: syscall
```

Comment:
juat to know, this is supposed to be a syscall number right?

Reply:
Number of name, should work in both cases